pom.xml

```diff
@@ -78,7 +78,8 @@
     <version.io.opentracing.contrib-opentracing-spring-jms>0.1.0</version.io.opentracing.contrib-opentracing-spring-jms>
     <version.io.opentracing.contrib-opentracing-spring-messaging>0.1.2</version.io.opentracing.contrib-opentracing-spring-messaging>
     <version.io.opentracing.contrib-opentracing-spring-rabbitmq>2.0.5</version.io.opentracing.contrib-opentracing-spring-rabbitmq>
-    <version.io.opentracing.contrib-java-jdbc>0.1.1</version.io.opentracing.contrib-java-jdbc>
+    <version.io.opentracing.contrib-java-jdbc>0.2.9-SNAPSHOT</version.io.opentracing.contrib-java-jdbc>
```

Comment:
We cannot update to JDBC 0.2.9. It uses OpenTracing 0.33.0 which is not compatible with the version used in this project.

Reply:
Blocked by https://github.com/opentracing-contrib/java-spring-cloud/issues/276